releases/release-4.0.16.md

```diff
@@ -0,0 +1,165 @@
+---
```

Comment:
Please add an entry to `release-timeline.md`, `release-notes.md`, `TOC.md`.

Reply:
> 

Fixed